Simplenote/src/main/java/com/automattic/simplenote/PreferencesFragment.java

```diff
@@ -116,6 +130,18 @@ public boolean onPreferenceClick(Preference preference) {
             }
         });
 
+        Preference deleteAppPreference = findPreference("pref_key_delete_account");
+        int preferenceLayout = ThemeUtils.isLightTheme(requireActivity()) ?
+                R.layout.preference_red_light : R.layout.preference_red_dark;
```

Comment:
Just a suggestion, I didn't test it to see if it works, why not specify the layout directly in `preferences.xml` using the `android:layout` option, to have a separate layout for dark them maybe you can define it in the `layout-night` folder, or define the colors in the `values-night` folder, and use the same layout.
WDYT?

Reply:
Yes, I think that makes the code much cleaner and removes code from the fragment that should be in the layout. It also makes it simpler and flexible to add more themes to the app. Fixed in c177fab7e3144f5469c27ed2a048105783961dcc.